cmd/api/app/routes/node/handler.go

```diff
@@ -0,0 +1,25 @@
+package node
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/karmada-io/dashboard/cmd/api/app/router"
+	"github.com/karmada-io/dashboard/cmd/api/app/types/common"
+	"github.com/karmada-io/dashboard/pkg/client"
+	"github.com/karmada-io/dashboard/pkg/resource/node"
+)
+
+func handleGetClusterNode(c *gin.Context) {
+	karmadaClient := client.InClusterClientForKarmadactlApiServer(c.Param("clustername"))
```

Comment:
maybe we need to check the existence of member cluster，after init karmada client, you can check with following implementation:
```go
_, err = client.ClusterV1alpha1().Clusters().Get(context.TODO(), clusterName, metav1.GetOptions{})
if err != nil {
    // member cluster not exist
}
```